src/CryptState.h

```diff
@@ -12,16 +12,12 @@
 #define AES_KEY_SIZE_BYTES  (AES_KEY_SIZE_BITS/8)
 
 #include "Timer.h"
+#include <string>
```

Comment:
Right now we're using Qt's types basically everywhere, but since a long-term goal of ours is to move away from Qt, I think it's fine if we start using standard types  in our code...